db.json

```diff
@@ -27,6 +27,10 @@
     {
       "name": "Wurst",
       "id": 7
+    },
+    {
+      "name": "",
+      "id": 8
```

Comment:
Do you need this entry to show something? If not I would delete the entry or revert the commit 😸

Reply:
Oh noooo. Too late. I already merged it. :(